Common/UnitDefinitions/VolumeFlow.json

```diff
@@ -261,6 +261,18 @@
         }
       ]
     },
+    {
+      "SingularName": "UkGallonPerHour",
+      "PluralName": "UkGallonsPerHour",
+      "FromUnitToBaseFunc": "x/791887.667",
+      "FromBaseToUnitFunc": "x*791887.667",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "gal (imp.)/hour" ]
```

Comment:
`gal (imp.)/h` would be more consistent with the library. Add both or just one of them?

Reply:
👍